libs/converter-nh-accuvote/src/proofing.ts

```diff
@@ -0,0 +1,535 @@
+      color: rgb(0, 1.0, 0),
+    });
+  }
+}
+
+function addBubbleAnnotationsToPdfPage(
+  page: PDFPage,
+  grid: TimingMarkGrid,
+  bubbles: BallotGridPoint[]
+): void {
+  const pageSize = newPdfSize(page.getWidth(), page.getHeight());
+
+  for (const bubble of bubbles) {
+    const bubbleCenterTemplatePoint = ballotGridPointToImagePoint(
+      grid.completeTimingMarks,
+      newBallotGridPoint(bubble.x, bubble.y)
+    );
+    const bubbleCenterPdfPoint = imagePointToPdfPoint(
+      pageSize,
+      grid.geometry.pixelsPerInch,
+      PDF_PPI,
+      bubbleCenterTemplatePoint
+    );
+
+    const bubbleMarkSize = 2;
+    page.drawLine({
+      start: {
+        x: bubbleCenterPdfPoint.x - bubbleMarkSize,
+        y: bubbleCenterPdfPoint.y - bubbleMarkSize,
+      },
+      end: {
+        x: bubbleCenterPdfPoint.x + bubbleMarkSize,
+        y: bubbleCenterPdfPoint.y + bubbleMarkSize,
+      },
+      thickness: 1,
+      color: rgb(1.0, 0, 0),
+    });
+    page.drawLine({
+      start: {
+        x: bubbleCenterPdfPoint.x - bubbleMarkSize,
+        y: bubbleCenterPdfPoint.y + bubbleMarkSize,
+      },
+      end: {
+        x: bubbleCenterPdfPoint.x + bubbleMarkSize,
+        y: bubbleCenterPdfPoint.y - bubbleMarkSize,
+      },
+      thickness: 1,
+      color: rgb(1.0, 0, 0),
+    });
+  }
+}
+
+function addContestOptionAnnotationsToPdfPage({
+  page,
+  grid,
+  gridPositions,
+  minimumDistanceBetweenBubbles,
+  optionTextConfig,
+  contestTextConfig,
+}: {
+  page: PDFPage;
+  grid: TimingMarkGrid;
+  gridPositions: GridPositionOption[];
+  minimumDistanceBetweenBubbles: BallotGridValue;
+  optionTextConfig: TextAppearanceConfig;
+  contestTextConfig: TextAppearanceConfig;
+}): void {
+  const pageSize = newPdfSize(page.getWidth(), page.getHeight());
+  const timingMarkTemplateSize = grid.geometry.timingMarkSize as ImageSize;
+  const timingMarkPdfSize = imageSizeToPdfSize(
+    grid.geometry.pixelsPerInch,
+    PDF_PPI,
+    timingMarkTemplateSize
+  );
+
+  for (const gridPosition of gridPositions) {
+    const bubbleCenterTemplatePoint = ballotGridPointToImagePoint(
+      grid.completeTimingMarks,
+      newBallotGridPoint(gridPosition.column, gridPosition.row)
+    );
+
+    const bubbleCenterPdfPoint = imagePointToPdfPoint(
+      pageSize,
+      grid.geometry.pixelsPerInch,
+      PDF_PPI,
+      bubbleCenterTemplatePoint
+    );
+
+    const contest = gridPosition.contestId;
+    const option = gridPosition.optionId;
+
+    const maximumOptionNameWidth = optionTextConfig.font.widthOfTextAtSize(
+      'Modest Option Name Length',
+      optionTextConfig.maxFontSize
+    );
+    const maximumContestNameWidth = contestTextConfig.font.widthOfTextAtSize(
+      'Modest Contest Name Length',
+      contestTextConfig.maxFontSize
+    );
+
+    const textColor = rgb(1.0, 1.0, 1.0);
+    const textBackgroundColor = rgb(0.5, 0.8, 0.5);
+    const textBackgroundOpacity = 0.8;
+    const textBackgroundPadding = 2;
```

Comment:
A little more luminance contrast between text color and background would improve readability (and also for write-in labels)

Reply:
Yeah, fair. It is pretty light. Updated:

![image](https://github.com/user-attachments/assets/fcb3091d-3e0f-4e3a-8d87-c28adc47e5d0)